sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusListener.cs

```diff
@@ -355,6 +355,8 @@ internal async Task ProcessMessageAsync(ProcessMessageEventArgs args)
             using (CancellationTokenSource linkedCts = CancellationTokenSource.CreateLinkedTokenSource(args.CancellationToken, _stoppingCancellationTokenSource.Token))
             {
                 var actions = new ServiceBusMessageActions(args);
+                _messagingProvider.ActionsCache.TryAdd(args.Message.LockToken, actions);
```

Comment:
I don't see where this is being removed from the cache when the message is done processing. Am I missing that?